### PR TITLE
feat: make access_log_settings nullable with dynamic block

### PR DESCRIPTION
**Pull request by MickVanDuijn**

### CODE REVIEW

**Comment by MickVanDuijn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #9** <a href="https://app.graphite.dev/github/pr/skyleague/aws-rest-api/9" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/aws-rest-api/9?utm_source=stack-comment).

**Review by pevisscher — APPROVED** [review]

**Comment by MickVanDuijn** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/aws-rest-api/9?utm_source=stack-comment).


Job Reference: bUFfnyyJ5OyWqV3R7QKf

**Pull request merged**
